cfpb/design-stories
# feat: Hero component

Closes #81 

A starting point for the variants of [Hero](https://cfpb.github.io/design-system/patterns/heroes) from the design system.  

## Changes

- Hero component that supports illustration, photograph, 50/50, and jumbo
- Can customize text and background colors, heading levels

## How to test this PR

1. Check out the [Hero stories](https://deploy-preview-83--cfpb-design-stories.netlify.app/?path=/story/components-hero--with-illustration)
2. Try resizing the browser window the see how the responsive changes feel
